variables.tf

```diff
@@ -97,6 +97,12 @@ variable "remote_state_consumer_ids" {
   description = "A set of workspace IDs set as explicit remote state consumers for this workspace"
 }
 
+variable "permissions_boundary_arn" {
```

Comment:
Can you also add permissions boundary to the user?

Reply:
Done